Microsoft.Dynamics365.UIAutomation.Sample/AzureDevOps/UCI/RegressionTests.cs

```diff
@@ -869,5 +887,455 @@ public void Regression_EntityControls_GetValue_SetValue_All()
             }
         }
 
+        [TestCategory("Regression")]
+        [TestCategory("Controls")]
+        [TestCategory("QuickCreate")]
+        [TestMethod]
+        public void Regression_QuickCreateControls_GetValue_SetValue_All()
+        {
+            var telemetry = new Microsoft.ApplicationInsights.TelemetryClient { InstrumentationKey = _azureKey };
```

Comment:
This a really long test. Can you please at least split it in methods instead of regions?